client.go

```diff
@@ -21,9 +21,10 @@ type Client interface {
 	SetActiveByPort(port int) error
 	SetActiveById(string) error
 
-	ListDeploys() ([]*Deploy, error)
+	ListDeploys(regex string) ([]*Deploy, error)
```

Comment:
It seems likely that this, very soon, will turn into a rich structure (eg, we might filter with a regex, or filter all deploys between a date range etc.)

This is best represented via a union type so
```
type DeployFilter interface {}

type RegexFilter struct {
   Regex string
}

// later, as an example
type DateRangeFilter struct {
   createdBefore int32
   createdAfter int32
}

ListDeploys(filter DeployFilter)...
```